src/state/transactions/reducer.ts

```diff
@@ -77,5 +79,12 @@ export default createReducer(initialState, (builder) =>
       } else if (transactions[chainId]?.[hash].type === 'limit-order-cancellation') {
         confirmOrderCancellation(chainId, receipt.from, hash, receipt.status !== 0)
       }
+    })
+    .addCase(resetUserState, (transactions, { payload: { chainId } }) => {
+      if (transactions[chainId]) {
+        transactions[chainId] = {}
+      }
+      const lsOrderKeys = getLocalStorageItemKeys(LS_ORDERS)
+      lsOrderKeys.forEach((lsOrderKey) => window.localStorage.removeItem(lsOrderKey))
```

Comment:
Why do we need to remove limit order localstorage? Isn't it store by account?

Reply:
Yes it is stored per account but we need to remove them to prevent other users to see the previous user’s actions on the same device by checking local storage